api/serverpb/server.proto

```diff
@@ -314,6 +326,10 @@ service Server {
       post: "/v1/Settings/Change"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "super puper method summary!"
+      description: "Awesome method description"
+    };
```

Comment:
Method summary and description

p.s. Summaries already present in our docs

Reply:
Aren't descriptions too? Try to add another sentence (ending it with a dot) to the comment.

Reply:
`/v1/readyz` has such comment, but on swagger UI it's visualized as the summary
```
  // Readiness returns an error when some PMM Server component is not ready yet or is being restarted.
  // It can be used as for Docker health check or Kubernetes readiness probe.
  rpc Readiness(ReadinessRequest) returns (ReadinessResponse) {
    option (google.api.http) = {
      get: "/v1/readyz"
    };
  }
```

![Screen Shot 2021-02-11 at 12 36 52](https://user-images.githubusercontent.com/21174514/107626090-f3a36300-6c65-11eb-8d5b-c2236c1bdf89.png)